.ci/download-deps.sh

```diff
@@ -3,7 +3,7 @@
 
 function get_deps() {
 
-	export TSS_VERSION=${TPM2_TSS_VERSION:-3.1.0}
+	export TSS_VERSION=${TPM2_TSS_VERSION:-3.2.0}
```

Comment:
Ha, I woke up last night and was like, I bet @idesai didn't bump the TSS version.